gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
was told by the gradle team that it's better to use -bin rather than -all

Reply:
Why this? IIRC the `-bin` doesn't come with sources. Therefore, you will miss on some features (i.e. cmd+b inside the `.gradle` files and so on).

Reply:
I recall this being an issue as well.

Reply:
ah, right, I guess they meant this probably for the apps that usually don't do much gradle logic/plugins, but for us it'd make sense to have sources indeed.

Reply:
Yup 👍  thanks for clarifying